js/templates/modals/dialog.html

```diff
@@ -0,0 +1,17 @@
+<div class="<%= ob.titleClass %>">
+  <h1><%= ob.title %></h1>
+</div>
+<div class="<%= ob.messageClass %>">
+  <p><%= ob.message %></p>
+  <% if (ob.buttons.length) { %>
+  <div class="buttonStrip row">
+    <% ob.buttons.forEach((btn, i) => { %>
+    <%
+      // pending designs, may need to give certain buttons special classes depending
+      // on what position they are in (e.g. first, last...)
+    %>
+      <a class="btn colPrimary colBorder js-<%= btn.fragment %>" data-event-name="<%= btn.fragment %>"><%= btn.text %></a>
```

Comment:
The color classes need to be updated here to clrP and clrBr. I shortened the class names in the last PR.